RELEASING.md

```diff
@@ -0,0 +1,29 @@
+# How to release
+
+## No changes at Java dependencies
+
+1. Edit `pom.xml` to increase the version
+2. Commit the change and create a PR
+3. Get the PR accepted
+4. Generate a release at https://github.com/openSUSE/subscription-matcher/releases/new (name will be vX.YY where X.YY is the version you have at pom.xml)
+5. Proceed to your working copy of systemsmanagement:Uyuni:Master and the package subscription matcher
+6. Run `SMVER=X.YY; wget https://github.com/openSUSE/subscription-matcher/archive/v${SMVER}.tar.gz && tar -xzf v${SMVER}.tar.gz && mv subscription-matcher-${SMVER} subscription-matcher && tar -czf subscription-matcher.tar.gz subscription-matcher && rm -rf subscription-matcher v${SMVER}.tar.gz` (replace X.YY with the version you have at pom.xml)
+7. Change the version at subscription-matcher.spec
+8. Run `osc vc` and add changes
+9. Run `osc build` to check that the package is building
+9. Commit the changes
+10. Copypac the package to all the Devel:Galaxy:Manager:X.Y projects with maintenance
```

Comment:
It might be nice to have the exact command here (and below for other copypac's) so that they can be copied and pasted.